### PR TITLE
create helper.js for helpful function

### PR DESCRIPTION
**Pull request by phodal**

### CODE REVIEW

**Comment by mcollina** [comment]
In order to accept this into the project, I need you to sign the [Eclipse CLA](https://www.eclipse.org/legal/CLA.php). 
To do that (quoted from the [FAQ](https://www.eclipse.org/legal/clafaq.php)):
> Log into the [Eclipse projects forge](https://projects.eclipse.org/user/login/sso) (you will need to create an account with the Eclipse Foundation if you have not already done so); click on "Contributor License Agreement"; and Complete the form. Be sure to use the same email address when you register for the account that you intend to use on Git commit records.

File: lib/coap.js [hunk 1]
```diff
@@ -32,16 +32,16 @@ function CoAP(opts, done) {
   this._persistence = opts.ponte.persistence;
   this._broker = opts.ponte.broker;
   this._ponte = opts.ponte;
-  
-  if (typeof opts.authenticate === "function") {
+
+  if (helper.isFunction(opts.authenticate)) {
```
Comment (mcollina) [inline lib/coap.js]: while we are at it, do something like:

```js
var isFunction = require("helper").isFunction;

if (isFunction(opts.authenticate)) {
  // ...
}
```

Comment (phodal) [inline lib/coap.js]: any plan for this project?

Comment (phodal) [inline lib/coap.js]: Hi, I already  sign the Eclipse CLA, but no work.

Comment (mcollina) [inline lib/coap.js]: can you please do a dummy commit, or do an `--amend` changing something silly? I do not think the hook gets called when closing/opening.

What Ponte is missing from a QEST perspective is an UI. Would you like to contribute?
I am working making better all the pubsub beneath it.

**Comment by mcollina** [comment]
Can you please sign off the commit? https://dev.eclipse.org/eclipse-webhook/services/status_details.php?id=54d0ebe3a54a5

**Comment by phodal** [comment]
Hi @mcollina, just click " confirm the origin of the submission" ?

**Comment by mcollina** [comment]
You should amend the commit with git commit -s --amend, and push force
Il giorno sab 4 lug 2015 alle 13:57 Fengda Huang <notifications@github.com>
ha scritto:

**Pull request closed without merging**